docs/src/home.md

```diff
@@ -1,160 +1,270 @@
+```@example home1
+1 + 2
+```
+and
+```@example home1
+@reaction_network begin
+    (p,d), 0 <--> X
+end
+```
+However, in some situations (e.g. when output is extensive, or irrelevant to what is currently being described) we have disabled this, e.g. like here:
+```@example home1
+1 + 2
+nothing # hide
+```
+and here:
+```@example home1
+@reaction_network begin
+    (p,d), 0 <--> X
+end
+nothing # hide
+```
 
+## [Installation](@id doc_home_installation)
+Catalyst is an officially registered Julia package, which can be installed through the Julia package manager:
 ```julia
 using Pkg
 Pkg.add("Catalyst")
 ```
 
-To solve Catalyst models and visualize solutions, it is also recommended to
-install DifferentialEquations.jl and Plots.jl
+Many Catalyst features require the installation of additional packages. E.g. for ODE-solving and simulation plotting
 ```julia
-Pkg.add("DifferentialEquations")
+Pkg.add("OrdinaryDiffEq")
 Pkg.add("Plots")
 ```
+is also needed.
 
-## Illustrative Example
-Here is a simple example of generating, visualizing and solving an SIR ODE
-model. We first define the SIR reaction model using Catalyst
-```@example ind1
-using Catalyst
-rn = @reaction_network begin
-    α, S + I --> 2I
-    β, I --> R
+A more throughout guide for setting up Catalyst and installing Julia packages can be found [here](@ref catalyst_for_new_julia_users_packages).
+
+## [Illustrative example](@id doc_home_example)
+
+#### [Deterministic ODE simulation of Michaelis-Menten enzyme kinetics](@id doc_home_example_ode)
+Here we show a simple example where a model is created using the Catalyst DSL, and then simulated as 
+an ordinary differential equation.
+
+```@example home_simple_example
+# Fetch required packages.
+using Catalyst, OrdinaryDiffEq, Plots
+
+# Create model.
+model = @reaction_network begin
+    kB, S + E --> SE
+    kD, SE --> S + E
+    kP, SE --> P + E
 end
+
+# Create an ODE that can be simulated.
+u0 = [:S => 50, :E => 10, :SE => 0, :P => 0]
+tspan = (0., 200.)
+ps = (:kB => 0.01, :kD => 0.1, :kP => 0.1)
+ode = ODEProblem(model, u0, tspan, ps)
+
+# Simulate ODE and plot results.
+sol = solve(ode)
+plot(sol; lw = 5)
 ```
-Assuming [Graphviz](https://graphviz.org/) and is installed and *command line
-accessible*, the network can be visualized using the [`Graph`](@ref) command
-```julia
-Graph(rn)
-```
-which in Jupyter notebooks will give the figure
 
-![SIR Network Graph](assets/SIR_rn.svg)
+#### [Stochastic jump simulations](@id doc_home_example_jump)
+The same model can be used as input to other types of simulations. E.g. here we instead perform a 
+jump simulation
+```@example home_simple_example
+# Create and simulate a jump process (here using Gillespie's direct algorithm).
+using JumpProcesses
+dprob = DiscreteProblem(model, u0, tspan, ps)
+jprob = JumpProblem(model, dprob, Direct())
+jump_sol = solve(jprob, SSAStepper())
+jump_sol = solve(jprob, SSAStepper(); seed = 1234) # hide
```

Comment:
For these, I am not linking the plot image (we could use this though). However, I use the same seed which was used to generate them (so ideally the images should appear the same on both pages, which I think would be good). 

I do not have to add a stable rng to the jump simulation here, right?